src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/binary_convolution.cpp

```diff
@@ -4,16 +4,17 @@
 
 #include <vector>
 
-#include "single_layer_tests/binary_convolution.hpp"
+#include "single_opt_tests/binary_convolution.hpp"
```

Comment:
`single_op_tests`. Please rename all folders.. What is `opt`??

Reply:
oh.. my fault.
I will change it